internal/api/offlinecheck/container.go

```diff
@@ -89,7 +89,7 @@ func LoadBinary(bytes []byte, db map[string]*ContainerCatalogEntry) {
 	}
 }
 
-func IsCertified(registry, repository, tag, digest string) bool {
+func (checker OfflineChecker) IsContainerCertified(registry, repository, tag, digest string) bool {
```

Comment:
You don't use `checker` here.  It probably doesn't need to be a function implemented in the struct?

Reply:
in offline one yes. but in the online version I do need it.

Reply:
I don't understand?

Reply:
Okay I think I understand now.  It's just organizational.  Sounds good.